proposals/ThirdPartyPackageManagers.md

```diff
@@ -32,3 +32,14 @@ The `name` attribute would specify the command-line package manager to be used,
 ## Framework Tag
 
 The `src` attribute can be a relative path or a URL. The `type` attribute specifies the type of framework that corresponds to the `engine` supported. In this case type `podspec` means that the lib should be installed using the `cocoapods` package manager, and `nuspec` for the `nuget` package manager.
+
+## Spec Argument
```

Comment:
We don't need to standardize this as it is platform specific. For iOS, it makes sense to have it separate as that is how it appears in the pod file. For gradle, it appears as `com.android.support:support-v13:23+`.